src/components/Auth/PasswordStrengthIndicator/PasswordStrengthIndicator.tsx

```
@@ -0,0 +1,38 @@
+  if (!isVisible) return null;
+
+  const checks = {
+    minLen: password.length >= 8,
+    lower: PASSWORD_REGEX.lowercase.test(password),
+    upper: PASSWORD_REGEX.uppercase.test(password),
+    num: PASSWORD_REGEX.numeric.test(password),
+    special: PASSWORD_REGEX.specialChar.test(password),
+  };
+
+  return (
+    <div role="status" aria-live="polite">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)